Compiler.Lexer/Compiler.Lexer/Tokenizer.cs

```diff
@@ -0,0 +1,403 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Compiler.Lexer
+{
+    public partial class Tokenizer
+    {
+        private CharStream Stream;
```

Comment:
Don't think you need "private" in local variable definition. They are private by default. Less words make readability easier.